File: CPP/Day14.cpp
/*

Day 14: Taking a and b as user input, print the value of a^b (a raised to b).(Use for loop)


*/ 

//Program
#include <iostream>
using namespace std;
double power(double a, int b) {
    double result = 1.0;
    for (int i = 0; i < b; i++) {
        result *= a;
    }
    return result;
}

int main() {
    double a;
    int b;
    cout<<"Enter the value of a: ";
    cin >> a;
    cout<<"Enter the value of b: ";
    cin >> b;

    double result = power(a, b);
    cout << a << " raised to the power of " << b << " is: " << result << endl;
    return 0;
}
  

  
/*


Input & Output: 

Enter the value of a: 5
Enter the value of b: 4
5 raised to the power of 4 is: 625


*/

File: Python/Day14.py
"""

Day 14:	Taking a and b as user input, print the value of a^b (a raised to b).(Use for loop)


"""

#Program
def power(a, b):
    
    result = 1
    for i in range (0, b):
        result = result * a
    
    print(f"The value of {a} raised to {b} is: ", result)


a = float(input("Enter the number: "))
b = int(input("Enter the power: "))

power(a, b)



"""

Input and Output: 
Enter the number: 2
Enter the power: 4
The value of 2.0 raised to 4 is:  16.0

"""